clouddriver-sql-postgres/clouddriver-sql-postgres.gradle

```diff
@@ -0,0 +1,6 @@
+dependencies {
+  implementation project(":cats:cats-sql")
+  implementation project(":clouddriver-sql")
+
+  runtimeOnly "org.postgresql:postgresql:42.2.18"
```

Comment:
Now that pg is being added into other services, we should probably look to move this to `spinnaker-dependencies` so the various services can be aligned.

Reply:
Yeah. There's a bit more follow up work to this. For instance I can't see `SqlTaskRepositoryTest` being executed - at least on build. I wonder if that's on purpose.